lms/djangoapps/grades/models.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth.models import User
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+from xmodule_django.models import BlockTypeKeyField, CourseKeyField, UsageKeyField
+
+class PersistentSubsectionGradeModel(TimeStampedModel):
+    """
+    A django model tracking persistent grades at the subsection level.
+
+    TODO: intended query patterns are fluid at the moment, document them here as they're firmed up.
+    """
+
```

Comment:
This is one table where we should probably use big ints off the bat, so we don't have to worry about running out of primary keys. An example of this in edx-platform is `StudentModuleHistoryExtended`:

```python
    id = UnsignedBigIntAutoField(primary_key=True)  # pylint: disable=invalid-name
```